validations/src/validations.rs

```diff
@@ -743,8 +743,12 @@ pub fn validate_commit_transaction(
     }
 
     let dr_output = &dr_state.data_request;
+    let pkh = co_tx.body.proof.proof.pkh();
 
     // A value transfer output cannot have zero value
```

Comment:
And may we should add as comment that it should also have the same PKH as output.